pyproject.toml

```diff
@@ -1,8 +1,9 @@
 [tool.poetry]
 name = "GamestonkTerminal"
 version = "0.1.0"
-description = ""
+description = "The next best thing after Bloomberg Terminal"
 authors = ["Didier Rodrigues Lopes"]
+readme = "README.md"
```

Comment:
@DidierRLopes Good start for now. I think a lot of other options are supported but it shouldn't make much of a difference.